.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: '3 3 * * 1'
```

Comment:
Could you remove the `cron` part, please? I'd hate to have this running by it's own.

Reply:
Done and repushed.

(I find the cron jobs useful to detect breakage that is related to dependencies or the changes CI environment. AS this project does not have any third-party dependencies, this seems to be less important here.)